docs/quickstart.md

```diff
@@ -165,6 +142,26 @@ STATUS: DEPLOYED
 
 ## 3. Use Helm to install Coherence
 
+Oracle Coherence docker image will be pulled by the Helm chart from
+[Oracle Container Registry](https://container-registry.oracle.com).
+
+And to be able to pull the image, you will need to
+
+a) Log in to [OCR](https://container-registry.oracle.com) and
+   accept the terms and conditions to download Coherence images.
+
+b) Create an image pull secret with your OCR credentials, and and tell
+   Kubernetes to use that secret when pulling the image.
+
+```bash
+$ kubectl create secret docker-registry sample-coherence-secret \
```

Comment:
Instead of "sample-coherence-secret", how about "oracle-container-registry-secret" ?